### PR TITLE
Supporting pillar based channel configuration

### PR DESCRIPTION
**Pull request by rawkode**

### CODE REVIEW

**Comment by rawkode** [comment]
This is pending some work on the repomaker. Please do not merge just yet.

**Comment by rawkode** [comment]
This is ready to merge :+1:

File: .kitchen.yml [hunk 1]
```diff
@@ -19,16 +19,30 @@ provisioner:
...
       '*':
         - extenda-repo
-  pillars:
-    top.sls:
-      base:
-        "*":
-          - vault
   grains:
-    customer: test-customer
+    customer: auc
```
Comment (sclausson) [inline .kitchen.yml]: @rawkode Why are you changing the default customer to "Auc" instead of "test-customer"?

Comment (rawkode) [reply .kitchen.yml]: This was a change we made during a call, for testing, that shouldn't be here. Updating now :+1:

**Comment by rawkode** [comment]
Sorted :+1:

**Pull request merged**